--- a/arch/xtensa/boot/dts/esp32s3.dtsi
+++ b/arch/xtensa/boot/dts/esp32s3.dtsi
@@ -8,7 +8,7 @@
 	interrupt-parent = <&pic>;
 
 	chosen {
-		bootargs = "earlycon=esp32s3uart,mmio32,0x60000000,115200n8,40000000 console=ttyS0,115200n8 debug rw root=mtd:rootfs no_hash_pointers ";
+		bootargs = "earlycon=esp32s3uart,mmio32,0x60000000,115200n8,40000000 console=ttyS0,115200n8 rw root=mtd:rootfs no_hash_pointers ";
 	};
 
 	aliases {
