--- a/loginutils/adduser.c
+++ b/loginutils/adduser.c
@@ -287,2 +287,2 @@
 		/* set the owner and group so it is owned by the new user,
-		 * then fix up the permissions to 2755. Can't do it before
+		 * then make the home private. Can't do it before
@@ -290 +290 @@
-		int mkdir_err = mkdir(pw.pw_dir, 0755);
+		int mkdir_err = mkdir(pw.pw_dir, 0700);
@@ -313 +313 @@
-		 || chmod(pw.pw_dir, 02755) != 0 /* set setgid bit on homedir */
+		 || chmod(pw.pw_dir, 0700) != 0
